.rtd-environment.yml

```diff
@@ -3,4 +3,5 @@ dependencies:
   - gsl
   - pip
   - pip:
-    - -e .[docs]
+    - astropy  @ git+https://github.com/astropy/astropy.git
```

Comment:
need to figure out a better way of setting astropy >= 4.3

Reply:
@adrn, any suggestions?

Reply:
Fine to pin this as is for now. I want to hold off on requiring >= 4.3 until gala v1.5 (next release is v1.4, hopefully within the next few weeks). By then, 4.3 should be out and we can change this to astropy>=4.3